apps/meteor/app/ui-message/client/ActionButtonSyncer.ts

```diff
@@ -19,6 +20,18 @@ export const addButton = (button: IUIActionButton): void => {
 		case UIActionButtonContext.MESSAGE_BOX_ACTION:
 			MessageBox.onAdded(button);
 			break;
+		case UIActionButtonContext.USER_DROPDOWN_ACTION:
+			const { appId, actionId, labelI18n, context } = button;
+			AccountBox.addItem({
+				...button,
+				name: button.labelI18n,
+				appId,
+				actionId,
+				labelI18n,
+				context,
+				isAppButtonItem: true,
+			});
```

Comment:
Please extract this block and its remove counterpart to its own file in the same way the other buttons are organized